### PR TITLE
Add "Glossary" section to UG

### LINKED ISSUE #285: [PE-D][Tester E] Unclear target audience and purpose
Tenant list vs Customer list

There is also client, tenant and customer. Are they all referring to the same group of people?

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: tzuennn/ped#1

### LINKED ISSUE #290: [PE-D][Tester B] [UG] Lack of Glossary
No glossary in UG to explain the terms used throughout the UG. Users who may not understand the terms are unable to look up on the UG.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: PresleyChew/ped#4

### PR DESCRIPTION
**Pull request by RobotWizzard**
Resolves #290.
Resolves #285 by adding "customer list" to the glossary.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1016,3 +1015,40 @@ the acceptable range). Therefore, edit the data file only if you are confident t
...
+
+* **Client**: A landlord whose properties are managed by you (a letting agent).
+
+* **Letting agent**: An individual that facilitates agreements between tenants and landlords to rent properties.
+
+* **Customer list**: The list of tenants who are renting a property.
+
+* **Rent due date**: The date ('dd' in `dd/mm/yyyy`) in each month when monthly rent is collected from tenants.
+    Ranges between `1` - `31`, without leading zeros.
+
```
Comment (gohsl99) [inline docs/UserGuide.md:1033]: Client, customer list and rent due date should explain in "notes about this user guide".
Letting agent may not be needed as the reader for this user guide is already a letting agent.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -115,15 +114,15 @@ for the team of AddressBook-Level 3.
...
-- The **customer** (as part of the rental information) referred to in this user guide is the tenant of the property managed by the client.
+- The **customer** (as part of the rental information) referred to in this user guide is the tenant of the property owned by the client.
 - **Monthly rent** is equivalent to **monthly rent amount**.
   - Both are referring to the monthly rent amount for the property managed by your client.
 - **Deposit** is equivalent to **deposit amount**.
-  - Both are referring to the security deposit amount for the property managed by your client, paid by the tenant at the start of the lease.
+  - Both are referring to the security deposit amount for the property owned by your client, paid by the tenant at the start of the lease.
 - **Customer list** is equivalent to **customers**.
-  - Both are referring to the name(s) of the tenant(s) for the property managed by your client.
+  - Both are referring to the name(s) of the tenant(s) for the property owned by your client.
```
Comment (gohsl99) [inline docs/UserGuide.md:125]: Client may or may not "owned" the property, see what are others' views on this.

File: docs/UserGuide.md (hunk 1 above)
Comment (RobotWizzard) [reply docs/UserGuide.md:1033]: They are already included in "notes about this user guide". They are included in glossary for completion.

File: docs/UserGuide.md (hunk 2 above)
Comment (winstonlimjy) [reply docs/UserGuide.md:125]: Owned is fine, as security deposit are usually given to landlords/owners.

**Pull request merged**